<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BwStatsWidget</class>
 <widget class="QWidget" name="BwStatsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="BWGraph" name="bwgraph_BW" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="informationSelector_GB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="informationSelector_GBHLayout">
      <item>
       <widget class="QLabel" name="friend_Label">
        <property name="text">
         <string>Friend:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="friend_CB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="type_Label">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="updn_CB">
        <item>
         <property name="text">
          <string>Up</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Down</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="service_Label">
        <property name="text">
         <string>Service:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="service_CB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Unit_Label">
        <property name="text">
         <string>Unit:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="unit_CB"/>
      </item>
      <item>
       <widget class="QCheckBox" name="logScale_CB">
        <property name="text">
         <string>Log scale</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="buttonHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BWGraph</class>
   <extends>QWidget</extends>
   <header location="global">gui/statistics/BWGraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
